cmd/helm/install.go

```diff
@@ -267,6 +267,10 @@ func (i *installCmd) run() error {
 		return prettyError(err)
 	}
 
+	if chartRequested.Metadata.Deprecated {
+		fmt.Fprintln(os.Stdout, "WARNING: This chart is deprecated")
```

Comment:
This should be on stderr, not stdout. That way the error message is not part of the output when it’s piped to another program.